extensions/data-plane/data-plane-http/src/main/java/org/eclipse/edc/connector/dataplane/http/pipeline/HttpRequestParams.java

```diff
@@ -32,7 +33,7 @@ public class HttpRequestParams {
     private String method;
     private String baseUrl;
     private String path;
-    private String queryParams;
+    private final Map<String, String> queryParams = new HashMap<>();
```

Comment:
I think the correct signature for this field is `Map<String, List<String>>`, as every param could have multiple occurrences.

Reply:
Not sure it's useful, as `HttpUrl.Builder.addQueryParameter` only supports unique value.

Reply:
no, it handles query params as a list, so there can be multiple values for the same key: https://github.com/1184893257/okhttp/blob/1149fa6233442eb686d5e8a363e6cf37deb52ed9/okhttp/src/main/java/com/squareup/okhttp/HttpUrl.java#L808